deny.toml

```diff
@@ -0,0 +1,39 @@
+
+# only check for the targets we currently publish
+targets = [
+  { triple = "x86_64-apple-darwin" },
+  { triple = "x86_64-unknown-linux-gnu" },
+  { triple = "x86_64-unknown-linux-musl" },
+  { triple = "x86_64-pc-windows-msvc" },
+]
+
+[advisories]
+vulnerability = "deny"
+unmaintained = "deny"
+notice = "deny"
+unsound = "deny"
+ignore = []
+
+[bans]
+multiple-versions = "deny"
+deny = []
+
+[sources]
+unknown-registry = "deny"
+unknown-git = "deny"
+allow-git = []
+
+[licenses]
+unlicensed = "deny"
```

Comment:
We should allow unlicensed code: it's public domain so it's compatible with any license.

Reply:
I disagree. All crates should specify a license. if they don't specify, copyright and licensing is in a mess, not knowing how to apply.

Instead they should use something like `WTFPL` or `UNLICENSE`

Reply:
Oh, I assumed that was unlicense. If it's lacking a license, then yes, we should deny it.